src/results/ResultsContainer.tsx

```diff
@@ -128,13 +128,13 @@ export class Results extends Component<ResultsProps, ResultsContainerState> {
     if (isFetching) {
       return <h3>Loading...</h3>;
     }
-    const info = infoMappings[namespace];
+    const { name, links, info } = infoMappings[namespace];
```

Comment:
I think instead of:

    const { name, links, info } = infoMappings[namespace];

There should be either:

     const { info } = infoMappings[namespace];
     const { name, links } = info;

or:

     const { name, links } = infoMappings[namespace].info;

Reply:
No it's actually fine - the object returned by `infoMappings[namespace]` contains ` name, links, info`, see `infoMappings.tsx`